tests/util/fake_statsd_server.py

```diff
@@ -80,9 +80,12 @@ def _run_server(self):
 
             sock.bind(socket_path)
 
+            if self.debug:
+                print("Listening via UDS on", socket_path)
+
             # We are using ctypes for shmem so we have to use a consistent
             # datatype across Python versions
-            if is_p3k():
+            if sys.version_info[0] > 2:
```

Comment:
why don't we use the common utility function?

Reply:
I mentioned it briefly in the PR description but we need to do some path mangling to do the import (`datadog.util.compat`) if I used the util since the script will run with the `sys.path` set to `datadogpy/tests/util` and not just `datadogpy` when run from CLI. It seemed like a simple conversion vs doing some gymnastics with the `sys.path.append` that would also be fragile.

Reply:
Can it be that you are running the server with different python than the one used to run tests? The other possibility would be running the script as `python tests/util/fake_statsd_server.py` in the virtualenv where `datadog` is installed.

Reply:
No, this is just a general issue of python importing. When you use the shebang line (`#!/...`), the initial "root" `sys.path` is set to the parent dir of the script itself (not the cwd). Normally this isn't a problem for third-party imports but if you're importing something from the same filesystem tree, you then have to change paths as `os.path.join(sys.path, "datadog")` doesn't exist from that starting dir. Things get even more complicated here as we need something from a path that has to go back up the tree out of the `tests` directory.

To work around this issue and to make it work on both regular tests and as a runnable script, I believe that there's only a few options that can be used:
- Remove dependencies from the parent paths (the approach I took here)
- Append parent path manually to `sys.path` (fragile)
- Mangle the `PYTHONPATH` variable (fragile, bad UX)
- Use a conditional `from .... import something` surrounded by `try/except ImportError` block to account for two different ways to run it (multiple logic paths, fragile, might not even work when run as a standalone script)

Reply:
The alternative would be registering the script in `console_scripts` (or custom one and load it with `pkg_resources`) possibly guarded with an optional `extras` or a condition.

We can more forward with current implementation as the "cure" for parent import issue is easy in this case.

Reply:
> The alternative would be registering the script in console_scripts (or custom one and load it with pkg_resources) possibly guarded with an optional extras or a condition.

That's true for an installed package but wouldn't work if someone just checked out the repo and tried to run it I think.

> We can more forward with current implementation as the "cure" for parent import issue is easy in this case.

👍 Do you mind approving the PR or is something else in need of fixing?